app/screens/InfoScreen/SponsorCard.tsx

```diff
@@ -29,6 +29,34 @@ export type TierLevelProps =
 
 export type SponsorCardProps = CommonProps & TierLevelProps
 
+function maxImageDimensions(tier: "Platinum" | "Gold" | "Silver" | "Bronze"): {
+  maxWidth: number
+  maxHeight: number
+} {
+  switch (tier) {
+    case "Platinum":
+      return {
+        maxWidth: 280,
+        maxHeight: 60,
+      }
+    case "Gold":
+      return {
+        maxWidth: 240,
+        maxHeight: 48,
+      }
+    case "Silver":
+      return {
+        maxWidth: 135,
+        maxHeight: 26,
+      }
+    default: // Bronze and Other(?)
```

Comment:
For some reason, I also have "Other" sponsors. I choose this route so that we would always have dimensions and I could run this function once inside of the component.

Reply:
not 100% sure what the "Other" tier is. will check with Justin and Gant

Reply:
I just figured I would account for it and make sure we had a size for it either way.

Reply:
so it looks like the if you filter by `is-a-current-sponsor`, we won't have an "Other". so basically we only have 4 tiers (`Platinum`, `Gold`, `Silver`, `Bronze`)

Reply:
that said, maybe instead of a `default`, lets exhaust our switch only and use `case "Bronze"`. thoughts?

Reply:
Had an idea and have exhausted the switch 😄